src/Halogen/VDom/Driver.purs

```diff
@@ -91,8 +91,8 @@ mkSpec handler renderChildRef document =
 
     patch
       :: EFn.EffectFn2 (WidgetState slots action)
-            (ComponentSlot slots Aff action)
-            (V.Step (ComponentSlot slots Aff action) DOM.Node)
+        (ComponentSlot slots Aff action)
+        (V.Step (ComponentSlot slots Aff action) DOM.Node)
```

Comment:
I suppose this is because of the lack of a `forall` or `->`.

Reply:
This is fixed in the 0.5 branch of purs-tidy.

Reply:
I'm still seeing it happen with 0.5.1, but maybe I'm accidentally hooking in to an older version.

Reply:
Working better now that this is updated to 0.5.1, thanks!